mxnet/buildspec.yml

```diff
@@ -54,28 +54,28 @@
       docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
       context:
         <<: *TRAINING_CONTEXT
-    BuildMXNetCPUTrainPy2DockerImage:
-      <<: *TRAINING_REPOSITORY
-      build: &MXNET_CPU_TRAINING_PY2 false
-      image_size_baseline: 4899
-      device_type: &DEVICE_TYPE cpu
-      python_version: &DOCKER_PYTHON_VERSION py2
-      tag_python_version: &TAG_PYTHON_VERSION py27
-      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
-      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-      context:
-        <<: *TRAINING_CONTEXT
-    BuildMXNetGPUTrainPy2DockerImage:
-      <<: *TRAINING_REPOSITORY
-      build: &MXNET_GPU_TRAINING_PY2 false
-      image_size_baseline: 4899
-      device_type: &DEVICE_TYPE gpu
-      python_version: &DOCKER_PYTHON_VERSION py2
-      tag_python_version: &TAG_PYTHON_VERSION py27
-      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
-      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
-      context:
-        <<: *TRAINING_CONTEXT
+    # BuildMXNetCPUTrainPy2DockerImage:
```

Comment:
Can we remove these if we are not building py2 images

Reply:
removed!